src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,8 +1,37 @@
+import ReusableInput from "./inputComponent";
+import { useForm } from "react-hook-form";
+
 const Dashboard = () => {
   // Dashboard tab
   console.log("Dashboard");
 
-  return <div>Dashboard</div>;
+  const onSubmit = (data: FormData) => {
+    console.log(data);
+  };
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  return (
+    <>
+      <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
There is an error here but it is fine for now since this issue is only focused on the input itself and not the actual use of hook form